meta/conf/distro/include/ptest-packagelists.inc

```diff
@@ -112,6 +112,7 @@ PTESTS_PROBLEMS:append:riscv64 = "valgrind-ptest"
 #    numactl-ptest \ # qemu not (yet) configured for numa; all tests are skipped
 #    libseccomp-ptest \ #  tests failed: 38; add to slow tests once addressed
 #    python3-numpy-ptest \ # requires even more RAM and (possibly) disk space; multiple failures
+#    run-postinsts-ptest \ # depends upon bash-ptest which is non-deterministic by design
 
 PTESTS_PROBLEMS = "\
```

Comment:
What does this variable do?

Reply:
Each of the PTESTS\_\* variables appear to provide a list for telling OE-core what ptests to run.

In this case, PTESTS_PROBLEMS means that it is not a consistently passing tests that also completes in a consistent amount of time